sdk/qumulo/Azure.ResourceManager.Qumulo/api/Azure.ResourceManager.Qumulo.net8.0.cs

```diff
@@ -180,9 +217,4 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Qumulo.Models.QumuloUserDetails>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Qumulo.Models.QumuloUserDetails>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
-    public enum StorageSku
-    {
-        Standard = 0,
-        Performance = 1,
-    }
```

Comment:
well we should not remove a type - and according to the api changes here I could guess that we removed this in the new version because the property that is using this is changed to use string as its type.
We will need to mitigate it later - after we fix the above issues to reduce the amount of changes in this file.